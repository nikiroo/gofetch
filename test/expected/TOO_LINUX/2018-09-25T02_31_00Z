          DANS UN MOIS AURA LIEU LE FORUM PHP 2018 À PARIS         

  2018/09/25

  o News link: https://www.toolinux.com/?Voici-le-programme-du-Forum-PHP-2018
  o Source link: 


    Dans un mois aura lieu le Forum PHP 2018 à Paris
    
    mardi 25 septembre 2018
    
    Il  se  tiendra  les  jeudi  25  et vendredi 26 octobre dans la
    capitale  française. L’événement n’a jamais été aussi séduisant
    sur  affiche. Une trentaine de conférenciers, venus de tous les
    horizons,  vont  y  partager leur savoir et vous faire profiter
    de leur expérience.
    
    C’est  bon  :  après des semaines de travail, les organisateurs
    du  Forum  PHP  ont  accouché  d’un splendide programme, [1]qui
    vient d’être publié en ligne .
    
    Parmi  les  invités  de  l’année,  notons  Anthony  Ferrara  et
    Matthias  Noback. Ils proposeront respectivement “Microservices
    gone  wrong”  et  “Beyond  design  patterns  and  principles  -
    writing  good  OO  code”,  deux  conférences  qui inviteront le
    public à réfléchir à sa façon de coder.
    
    Michelle  Sanver, autre membre très actif de la communauté PHP,
    vousfera  aussi  le  plaisir de nous rejoindre pour présenter “
    We got rid of management : we love it ! ”.
    
    De  nombreux  orateurs  francophones venus de l’étranger seront
    également  des  vôtres. Nuno Maduro parlera des applications de
    console  avec  Laravel  Zero et David Buchmann en dira plus sur
    bash  et  ses  pièges  - et tous les deux en français s’il vous
    plaît  !  Matthieu  Napoli répondra aux questions que vous vous
    posez  sur  Serverless  et  PHP et Pascal Martin vous dira tout
    sur la migration vers Kubernetes de la plateforme 6play.fr.
    
    Enfin,  un  programme  d’accompagnement permettra à de nouveaux
    orateurs  de passer derrière le pupitre de manière plus sereine
    grâce  aux  conseils  et  à l’écoute reçus durant les prochains
    mois de la part des conférenciers confirmés.
    
    La  billetterie passe aux tarifs de croisière : le pass 2 jours
    est   à   175€   pour  les  membres  AFUP,  275€  plein  tarif.
    L’événement  sera  probablement  complet  rapidement,  donc  ne
    tardez plus.
    
    Les liens
    
    [2]Le site du Forum PHP 2018
    
    [3]Le programme complet
    
    [4]La billetterie
    
    [5]
    
    
    
    [1] https://event.afup.org/en/forumphp2018/programme/
    
    [2] https://event.afup.org
    
    [3] https://event.afup.org/en/forumphp2018/programme/
    
    [4] https://afup.org/event/forumphp2018/tickets?_locale=fr
    
    [5]  https://www.toolinux.com/?Voici-le-programme-du-Forum-PHP-
    2018#forum



