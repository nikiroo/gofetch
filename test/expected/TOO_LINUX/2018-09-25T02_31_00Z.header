0Dans un mois aura lieu le Forum PHP 2018 à Paris	null/TOO_LINUX/2018-09-25T02_31_00Z		70
i  2018/09/25
i
i    Il  se  tiendra  les  jeudi  25  et vendredi 26 octobre dans la
i    capitale  française. L’événement n’a jamais été aussi séduisant
i    sur  affiche. Une trentaine de conférenciers, venus de tous les
i    horizons,  vont  y  partager leur savoir et vous faire profiter
i    de  leur expérience. C’est bon : après des semaines de travail,
i    les organisateurs du Forum PHP ont accouché (...)
i
