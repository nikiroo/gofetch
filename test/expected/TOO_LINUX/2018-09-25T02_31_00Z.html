<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>Dans un mois aura lieu le Forum PHP 2018 à Paris</h1>
	<div class='details'>(2018/09/25)</div>
	<br/>
    <ul>
        <li>News link: <a href='https://www.toolinux.com/?Voici-le-programme-du-Forum-PHP-2018'>https://www.toolinux.com/?Voici-le-programme-du-Forum-PHP-2018</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Dans un mois aura lieu le Forum PHP 2018 à Paris<br/><br/>mardi 25 septembre 2018<br/><br/>Il se tiendra les jeudi 25 et vendredi 26 octobre dans la capitale française. L’événement n’a jamais été aussi séduisant sur affiche. Une trentaine de conférenciers, venus de tous les horizons, vont y partager leur savoir et vous faire profiter de leur expérience.<br/><br/>C’est bon : après des semaines de travail, les organisateurs du Forum PHP ont accouché d’un splendide programme, [1]qui vient d’être publié en ligne .<br/><br/>Parmi les invités de l’année, notons Anthony Ferrara et Matthias Noback. Ils proposeront respectivement “Microservices gone wrong” et “Beyond design patterns and principles - writing good OO code”, deux conférences qui inviteront le public à réfléchir à sa façon de coder.<br/><br/>Michelle Sanver, autre membre très actif de la communauté PHP, vousfera aussi le plaisir de nous rejoindre pour présenter “ We got rid of management : we love it ! ”.<br/><br/>De nombreux orateurs francophones venus de l’étranger seront également des vôtres. Nuno Maduro parlera des applications de console avec Laravel Zero et David Buchmann en dira plus sur bash et ses pièges - et tous les deux en français s’il vous plaît ! Matthieu Napoli répondra aux questions que vous vous posez sur Serverless et PHP et Pascal Martin vous dira tout sur la migration vers Kubernetes de la plateforme 6play.fr.<br/><br/>Enfin, un programme d’accompagnement permettra à de nouveaux orateurs de passer derrière le pupitre de manière plus sereine grâce aux conseils et à l’écoute reçus durant les prochains mois de la part des conférenciers confirmés.<br/><br/>La billetterie passe aux tarifs de croisière : le pass 2 jours est à 175€ pour les membres AFUP, 275€ plein tarif. L’événement sera probablement complet rapidement, donc ne tardez plus.<br/><br/>Les liens<br/><br/>[2]Le site du Forum PHP 2018<br/><br/>[3]Le programme complet<br/><br/>[4]La billetterie<br/><br/>[5]<br/><br/><br/><br/>[1] https://event.afup.org/en/forumphp2018/programme/<br/><br/>[2] https://event.afup.org<br/><br/>[3] https://event.afup.org/en/forumphp2018/programme/<br/><br/>[4] https://afup.org/event/forumphp2018/tickets?_locale=fr<br/><br/>[5] https://www.toolinux.com/?Voici-le-programme-du-Forum-PHP-2018#forum
	</div>
<hr/>
</div>
</body>
